main.go

```diff
@@ -85,6 +85,10 @@ func main() {
 	if config.Execute != "" {
 		input, err = executeCommand(config)
 		if err != nil {
+			if input != "" {
+				// show the full error message output
+				printErrorFatal("Something went wrong", errors.New(input))
```

Comment:
What about replacing the line by this

```suggestion
				err = fmt.Errorf("%w %s", err, input)
```